api/src/environment.js

```diff
@@ -1,9 +1,20 @@
 const path = require('path');
 const logger = require('./logger');
 
-const { UNIT_TEST_ENV, COUCH_URL, BUILDS_URL } = process.env;
+const { 
+  UNIT_TEST_ENV,
+  COUCH_URL,
+  BUILDS_URL, 
+  // If set to true, PROXY_CHANGE_ORIGIN_ALL will override the values of PROXY_CHANGE_ORIGIN, 
+  // PROXY_CHANGE_ORIGIN_AUTH and PROXY_CHANGE_ORIGIN_CHANGES and set them to true.
```

Comment:
This comment is out of date now I think?

Reply:
Yes, thanks